_visual/typography.md

```diff
@@ -971,7 +971,7 @@ order: 01
   <p>This is a link which opens in a <a href="http://media.giphy.com/media/8sgNa77Dvj7tC/giphy.gif" target="_blank">new tab</a>.</p>
 
   <div class="usa-background-dark">
-    <p><a href="javascript:void(0)">This</a> is a text link on light background</p>
+    <p><a href="javascript:void(0)">This</a> is a text link on dark background</p>
```

Comment:
@carodew done in 8dd5c23.

Also added in periods bc they are complete sentences in 62daed3.